Formula/i/ircii.rb

```diff
@@ -1,9 +1,9 @@
 class Ircii < Formula
   desc "IRC and ICB client"
-  homepage "http://www.eterna.com.au/ircii/"
-  url "https://ircii.warped.com/ircii-20210314.tar.bz2"
-  mirror "https://deb.debian.org/debian/pool/main/i/ircii/ircii_20210314.orig.tar.bz2"
-  sha256 "866f2b847daed3d70859f208f7cb0f20b58c0933b2159f7ff92a68c518d393a9"
+  homepage "https://web.archive.org/web/20231024192652/http://eterna.com.au/ircii/"
```

Comment:
reaching out to the upstream about the site issue.

Reply:
We should add a comment explaining that it should be temporary and that we've reached out.

Reply:
just added an note, let me know if that works.